ros_buildfarm/templates/snippet/plot_plugin.xml.em

```diff
@@ -1,8 +1,13 @@
+@{
+import html
+}@
+
```

Comment:
This empty line should be removed since it shouldn't be part of the expanded template.

Reply:
We can't do something like:

```
          <description>@(import html; html.escape(plot.description))</description>
```

any suggestions?

Reply:
I think @dirk-thomas is just asking for the empty line 4 to be removed.